packages/admin/src/table/filterbar/filterBarPopoverFilter/FilterBarPopoverFilter.styles.tsx

```diff
@@ -0,0 +1,97 @@
+import { ButtonProps } from "@material-ui/core";
+import { Theme } from "@material-ui/core";
+import { makeStyles } from "@material-ui/styles";
+
+export type CometAdminFilterBarPopoverFilterClassKeys =
+    | "root"
+    | "fieldBarWrapper"
+    | "fieldBarWrapper--hasValues"
```

Comment:
@jamesricky do we have a naming convention for conditional classes like this one?

Reply:
BEM is soo 2010 :)

Reply:
I think the mui-way is like in this example: https://material-ui.com/guides/api/#css-classes (`'&$checked'`)

though I don't know how this technically works

Reply:
i know its an old approach but it was the first time something like that is used in the comet admin so i was not sure what would be the correct way :) i just gave it a try :D

Reply:
I think the right way to do this (and also the MUI way), is just with one class for the element in general and one additional class (non-BEM) for every state of the element. 

```js
| "fieldBarWrapper",            // Class, always set on element
| "fieldBarWrapperWithValues",  // Class, only set on element if there are values
```

This could then be styled, either like this:
```js
export const useStyles = makeStyles(
    () => ({
        fieldBarWrapper: {},
        fieldBarWrapperWithValues: {
            // styles with values set
        },
    })
);
```

or like this: 
```js
export const useStyles = makeStyles(
    () => ({
        fieldBarWrapper: {
            "&$fieldBarWrapperWithValues": {
                // styles with values set
            }
        },
        fieldBarWrapperWithValues: {},
    })
);
```

So basically just like it was, except without BEM.